docs/reference/extensions/django-framework.rst

```diff
@@ -17,6 +17,15 @@ server metrics.
     The Django extension is compatible with the ``bare``, ``ubuntu@22.04``
     and ``ubuntu@24.04`` bases.
 
+Django framework supports both synchronous and asynchronous. If you want
+asynchronous you have to add ``django-framework/async-dependencies``
+read more :ref:`django-framework/dependencies <django-sync-deps>`.
+
+If you define
+``django-framework/async-dependencies`` you can not use
+``django-framework/dependencies``. ``rockcraft pack`` will error if you try to
+use both at the same time.
```

Comment:
I would rephrase this paragraph to say something like
```
If you define
``django-framework/async-dependencies`` you can not
also include ``django-framework/dependencies``.
Running ``rockcraft pack`` will result in an error if you try to
use both at the same time.
```
Is it possible to include the error here?
(Same for Flask, although I won't add a comment there)